terraform/modules/kubernetes/kubernetes.tf

```diff
@@ -331,9 +494,9 @@ resource "kubernetes_cron_job" "sample_maker" {
               args = [
                 "--pushgateway", var.pushgateway,
                 "generate-ingestion-sample",
-                "--own-output", var.ingestion_bucket,
-                "--peer-output", var.test_peer_ingestion_bucket,
-                "--peer-identity", var.remote_peer_validation_bucket_identity,
+                "--peer-output", var.ingestion_bucket,
+                "--own-output", var.test_peer_ingestion_bucket,
+                "--own-identity", var.remote_peer_validation_bucket_identity,
```

Comment:
Out of curiosity, why did peer-output and own-output switch for the sample-maker deployment?

Reply:
While testing, I found that they were wrong, and were causing packet decryption errors because the packet encryption keys were getting mixed up. It presumably had been wrong for some time, presumably since some change I made hastily in December without testing it through a paired test env.